lib/galaxy/tools/toolbox/watcher.py

```diff
@@ -71,6 +71,8 @@ class ToolConfWatcher(object):
 
     def __init__(self, reload_callback, tool_cache=None):
         self.paths = {}
+        self.drop_on_next_loop = set()
+        self.drop_now = set()
```

Comment:
Shouldn't `drop_on_next_loop` and `drop_now` be local variables inside the `check()` method (like `hashes`)?

Reply:
Yes, good thinking!